integrationTests/consensus/testInitializer.go

```diff
@@ -426,6 +426,15 @@ func createConsensusOnlyNode(
 		node.WithConsensusType(consensusType),
 		node.WithBlackListHandler(&mock.BlackListHandlerStub{}),
 		node.WithBootStorer(&mock.BoostrapStorerMock{}),
+		node.WithRequestedItemsHandler(&mock.RequestedItemsHandlerStub{
```

Comment:
a stub should implement all these functions, so there would not be a need to redeclare them

Reply:
done